tests/module/mobject/graphing/test_bar_chart.py

```diff
@@ -0,0 +1,15 @@
+from manim.mobject.graphing.probability import BarChart
+
+
+def test_values_close_to_zero():
+    """Checks that BarChart supports values/heights close to zero without crashing if ."""
+    values = [1 / 10000 for _ in range(8)]
+    names = list(range(len(values)))
+    chart = BarChart(
+        values=values,
+        bar_names=["one", "two", "three", "four", "five"],
```

Comment:
Did you mean to pass `names` to `bar_names`?

Reply:
Yes, thank you, I will fix that.